root/lib/systemd/system/titus-metadata-proxy@.service

```diff
@@ -6,12 +6,16 @@ ConditionPathIsDirectory=/var/lib/titus-inits/%i/ns
 StartLimitIntervalSec=30
 StartLimitBurst=10
 
+After=titus-salt-generator.service
```

Comment:
@gabrielhartmann is the systemd expert on the team. Is this right? We have a "soft" dep on this thing starting.

Reply:
I believe `Wants` accomplishes the that.